packages/driver/cypress/fixtures/shadow-dom-button.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Any reason to have this be a separate file and not add it to `fixtures/shadow-dom.html`?

Reply:
There are 2 reasons:

1. `shadow-dom.html` creates shadow elements with `innerHTML` but it is a bit unintuitive in this case because we need to attach a listener in the constructor. We need to add another version of test element and it might make the fixture a bit cluttered.
2. As I said above, this fixture can be used for other failures like #17531.